README.md

```diff
@@ -162,6 +162,14 @@ $ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/aws-clo
 ```
 Remember to add `~/.guard/bin/` to your `$PATH`.
 
+Altenatively, you can install the latest version with [Homebrew](https://brew.sh/).
```

Comment:
Please correct the spelling - "Alternatively".

Reply:
Fixed. Thanks 🙇